src/app/countriesCard/Card.tsx

```diff
@@ -0,0 +1,36 @@
+import Image from "next/image"
+
+interface Propstype  {
+  flag : string,
+  alt : string,
+  name : string,
+  population : string,
+  region : string,
+  capital : string,
+}
+export default function CountriesCard (props : Propstype)  {
```

Comment:
This component will be used to render a SINGLE country card. Hence the name should reflect it. Something like CountryCard